src/Microsoft.Identity.Web/IdHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Reflection;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Identity.Web
+{
+    internal static class IdHelper
+    {
+        private static readonly Lazy<string> s_idWebVersion = new Lazy<string>(
+           () =>
+           {
+               string? fullVersion = typeof(IdHelper).GetTypeInfo().Assembly.FullName;
```

Comment:
does it need to be a `string?`  ?

Reply:
i was getting the null ref warning, so just returning an empty string if the version doesn't match (don't see how that's possible).

---
In reply to: [487406633](https://github.com/AzureAD/microsoft-identity-web/pull/572#discussion_r487406633) [](ancestors = 487406633)